### DISCUSSION TITLE
[deleted]

### DISCUSSION THREAD

**Discussion by edtomb** [community/community#120830] [Code Security] [answered]
[removed]

**Comment by midway** [comment]
This might come as a shock to you, but there are going to be many people in the world who have the same name as you.  There doesn't seem to be anything suspicious here.

**Reply by henri14871** [reply]
ethan i have a suspicion it is you dont lie to me

**Reply by midway** [reply]
So you downloaded and ran code you clearly didn't understand which was labeled as sample exploit code?

**Reply by edtomb** [reply]
No. It’s definitely not that. I could have installed the wrong hacked client for Minecraft(sorry Notch), or it’s nothing and my laptop cannot handle telemetry and cs2 at the same time. The more reasonable answer is that I got really unlucky in the username lottery. Glad I could put this out there either way. Thanks guys 😂

**Comment by samus-aran** [comment, marked as answer]
Hey @ethantomb.. I'm sure your reputation is a-ok 👌 😉 

The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Reply by edtomb** [reply]
Thank you! Just glad I got on top of this fire before it was too late! It’s quite possible that this is why I haven’t been getting any callbacks. Haha…
